schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-1

```diff
@@ -0,0 +1,192 @@
+              "additionalProperties": false
+            }
+          ]
+        },
+        "timestamps": {
+          "description": "Set of timestamps associated with the batch",
+          "type": "object",
+          "properties": {
+            "jobStarted": {
+              "description": "Time when the batch started being shredded",
+              "type": "string",
+              "format": "date-time"
+            },
+            "jobCompleted": {
+              "description": "Time when the batch shredding has been finished (and the message being prepared)",
+              "type": "string",
+              "format": "date-time"
+            },
+            "min": {
+              "description": "The earliest collector_stamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            },
+            "max": {
+              "description": "The latest collector_tstamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            }
+          },
+          "required": ["jobStarted", "jobCompleted", "min", "max"]
+        },
+        "processor": {
+          "description": "Identificator of a shredder sent the message",
+          "type": "object",
+          "properties": {
+            "artifact": {
+              "description": "Name of the artifact",
+              "type": "string",
+              "maxLength": 64
+            },
+            "version": {
+              "description": "Semantic Version of the artifact",
+              "type": "string",
+              "maxLength": 16
+            }
+          },
+          "required": ["artifact", "version"]
+        },
+        "count": {
+          "description": "Count of events in the batch, null means the count could not be calculated",
+          "type": ["object", "null"],
+          "properties": {
+            "good": {
+              "description": "Amount of good events in the batch",
+              "type": "integer",
+              "minimum": 0
+            }
+          }
+        }
+      },
+      "required": ["base", "compression", "typesInfo", "timestamps", "processor"],
+      "additionalProperties": false
+    },
+    "application": {
+      "description": "Loader's name and version",
+      "type": "string",
+      "maxLength": 128
+    },
+    "attempt": {
+      "description": "How many attempts were taken to load the batch",
+      "type": "integer",
+      "minimum": 0
+    },
+    "loadingStarted": {
+      "description": "Time when loader started loading (first attempt)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "loadingCompleted": {
+      "description": "Time when loader finished loading (should match the ingestion timestamp in manifest)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "recoveryTableNames": {
+      "description": "List of recovery table names created",
```

Comment:
> List of recovery table names created

Is it for tables _created_ by this batch, or tables _used_ by this batch?  In other words.... if a pipeline continually uses an invalid schema over several weeks, should it appear only once in the `load_succeeded_3` table, or should it appear for every batch that used the invalid schema?

Reply:
It is `created`, not `used`. If a pipeline keep using a broken schema, we expect to see the corresponding recovery table name once.

Reply:
> should it

Our discussions have been about reporting the creation if I remember right. On the other hand, stdout logs have both creation and usage separately.

If we want to change the semantic to `usage`, or add it, a batch could be loaded to the main table or a set of recovery table(s). In that case, we should have a `loadedTables` key.